.github/renovate.json

```diff
@@ -1,7 +1,9 @@
 {
+  "extends": [
+    "github>hmcts/.github//renovate/global",
+    "github>hmcts/.github//renovate/automerge-all"
+  ],
   "enabledManagers": ["helm-requirements","gradle-wrapper"],
```

Comment:
remove this? as we now extend from base?

Reply:
will do